src/docs-rs/maintenance.md

```diff
@@ -122,9 +122,18 @@ UPDATE queue SET attempt = 0 WHERE attempt >= 5 AND build_time > 'YYYY-MM-DD HH:
 
 ## Removing a crate from the website
 
-Sometimes it might be needed to remove all the content related to a crate from
-docs.rs (for example after receiving a DMCA). To do that, log into the server
-and run:
+The docs.rs application supports deleting all the documentation ever published
+of a crate (for example after receiving a DMCA), by running a CLI command. The
+people who currently have permissions to access the server and run it are:
+
+* docs.rs Team:
+  * [@pietroalbini](https://github.com/pietroalbini)
+  * [@jyn514](https://github.com/jyn514)
+* Infrastructure Team:
+  * [@Mark-Simulacrum](https://github.com/Mark-Simulacrum)
```

Comment:
```suggestion
* docs.rs Team:
  * [@jyn514](https://github.com/jyn514)
  * [@syphar](https://github.com/syphar)
  * [@Nemo157](https://github.com/nemo157)
* Infrastructure Team:
  * [@Mark-Simulacrum](https://github.com/Mark-Simulacrum)
  * [@pietroalbini](https://github.com/pietroalbini)
```
I worry that it doesn't make sense to hard-code this separately from https://github.com/orgs/rust-lang/teams/docs-rs ? Can we link there instead?